.github/workflows/CI-python.yml

```diff
@@ -40,57 +40,28 @@ jobs:
         with:
           python-version: ${{ matrix.pythonVersion }}
 
-      - uses: conda-incubator/setup-miniconda@v2
-        with:
-          auto-update-conda: true
-          python-version: ${{ matrix.pythonVersion }}
-
-      - if: ${{ matrix.operatingSystem != 'macos-latest' }}
-        name: Install pytorch on non-MacOS
-        shell: bash -l {0}
-        run: |
-          conda install --yes --quiet pytorch torchvision captum cpuonly -c pytorch
-
-      - if: ${{ matrix.operatingSystem == 'macos-latest' }}
-        name: Install Anaconda packages on MacOS, which should not include cpuonly according to official docs
-        shell: bash -l {0}
-        run: |
-          conda install --yes --quiet pytorch torchvision captum -c pytorch
-
-      - if: ${{ matrix.operatingSystem == 'macos-latest' }}
-        name: Install latest lightgbm from conda-forge for MacOS
-        shell: bash -l {0}
-        run: |
-          conda install --yes --quiet lightgbm -c conda-forge
-
       - name: Setup tools
-        shell: bash -l {0}
         run: |
           python -m pip install --upgrade pip
           pip install --upgrade setuptools
           pip install --upgrade "pip-tools<=7.1.0"
 
       - name: Install package dependencies
-        shell: bash -l {0}
         run: |
           pip install -r requirements-dev.txt
         working-directory: ${{ matrix.packageDirectory }}
 
-      - if: ${{ matrix.packageDirectory == 'rai_test_utils' }}
-        name: Install package extras
-        shell: bash -l {0}
+      - name: Install test extras
```

Comment:
shouldn't this object detection section be removed altogether here, since we are not installing torch?

Reply:
Good question, these dependencies only contain `requests` and `Pillow`. None of the `requirements-object-detection.txt` files contain torch, torch is always installed via conda. I added this step for tests to work across `rai_test_utils` and `erroranalysis` that utilizes methods from rai test utils in its tests. 

Considering rai test utils has common code to be used by other modules for testing + isn't part of any other builds, this step ensures that any non-torch tests (eg - loading image datasets) can run across all the other python modules.

Still, trying out another way (perhaps more elegant) that doesn't require this step section.